pyglmnet/pyglmnet.py

```diff
@@ -448,6 +451,23 @@ class GLM(BaseEstimator):
     verbose : boolean or int
         default: False
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> random_state = 1
+    >>> np.random.seed(random_state)
```

Comment:
as commented elsewhere, this is not a recommended approach. It leads to reproducibility issues